src/main/java/org/junit/experimental/theories/ParameterSignature.java

```diff
@@ -34,7 +34,7 @@
         map.put(b, a);
     }
     
-    public static ArrayList<ParameterSignature> signatures(Method method) {
+    public static List<ParameterSignature> signatures(Method method) {
```

Comment:
Not in public API signatures. If we have a public method that returns an `ArrayList` we can't change it to return a `List`:

    ArrayList<String> s = ParameterSignature.signatures(method); // woops!